query-languages/m/m-spec-values.md

```diff
@@ -568,14 +568,14 @@ The native type of record values is the intrinsic type `record`, which specifies
 
 A _table value_ is an ordered sequence of rows. A _row_ is an ordered sequence of column values. The table's type determines the length of all rows in the table, the names of the table's columns, the types of the table's columns, and the structure of the table's keys (if any).
 
-There is no literal syntax for tables. Several standard library functions are provided to construct table values. For example, `#table` can be used to construct a table from a list of row lists and a list of header names:
+Tables may be constructed using the `#table` intrinsic. 
```

Comment:
```suggestion
There is no literal syntax for tables. Several standard library functions are provided to construct table values. Tables may also be constructed using the `#table` intrinsic. 
```

Reply:
Thanks, @ehrenMSFT!

> There is no literal syntax for tables. Several standard library functions are provided to construct table values. 

For symmetry, if we add the above here, should we also add similar text to date, datetime, duration, etc., as the same concept applies to those types, as well?

Reply:
Yes, I think that would make sense. Slight tweak:
```suggestion
Although there is no literal syntax for tables, several standard library functions are provided to construct them. Tables may also be constructed using the `#table` intrinsic.
```
So for date/datetime/etc. this would become:

Although there is no literal syntax for dates, several standard library functions are provided to construct them. Dates may also be constructed using the `#date` intrinsic.

Reply:
Thanks. I'll work to apply that when I have the chance.

Question on the phraseology "`date`/`table`/etc. intrinsic`": Do you think it should be paired with the word "function"?

Example:
> Dates may also be constructed using the `#date` intrinsic function.

or

> Dates may also be constructed using the intrinsic function `#date`

Reply:
I'm fine with adding the word function for clarity. I like your second example ("intrinsic function #date").

Reply:
Thanks! Changes applied. What do you think?